packages/nextjs/src/app-router/server/auth.ts

```diff
@@ -131,7 +132,7 @@ export const auth: AuthFn = async () => {
   };
 
   return Object.assign(authObject, { redirectToSignIn, redirectToSignUp });
-};
+}) as AuthFn;
```

Comment:
Why do we need to cast here now?

Reply:
Same as my comment [here](https://github.com/clerk/javascript/pull/5689#discussion_r2067057156), but for this PR, it's just to make TS happy. The full context is in the Nextjs companion PR https://github.com/clerk/javascript/pull/5710